src/services/pcn-iptables/src/datapaths/Iptables_ChainSelector_dp.c

```diff
@@ -132,6 +132,8 @@ static int handle_rx(struct CTXTYPE *ctx, struct pkt_metadata *md) {
   #if _INGRESS_ALLOWLOGIC
       pcn_log(ctx, LOG_DEBUG, "INGRESS LOGIC PASS. No rules for INPUT and FORWARD, and default is ACCEPT. ");
       updateForwardingDecision(PASS_LABELING);
+      incrementDefaultCountersInput(md->packet_len);
+      _DEFAULTACTION_INPUT
```

Comment:
Not useful, if `_INGRESS_ALLOWLOGIC` is `true`, `_DEFAULT_ACTION_INPUT` will be replaced with [1]

[1] https://github.com/polycube-network/polycube/blob/106ece0232987bec295bda6419f1151aaf3473a8/src/services/pcn-iptables/src/modules/ChainSelector.cpp#L62